gcp-common/src/main/java/io/micronaut/gcp/credentials/GoogleCredentialsFactory.java

```diff
@@ -83,11 +83,11 @@ public GoogleCredentialsFactory(@NonNull GoogleCredentialsConfiguration configur
      * @return The {@link GoogleCredentials}
      * @throws IOException An exception if an error occurs
      */
-    @Requires(missingBeans = GoogleCredentials.class)
```

Comment:
why removing this requirement?

Reply:
It's somewhat redundant and odd that it was ever added here in the first place, and it was preventing the AOP interceptor from working (which may be a separate issue worth investigating). The documented and intended way to replace the framework-supplied `GoogleCredentials` being created by this factory is to use `gcp.credentials.enabled=false`.

Note that I can't think of any good reason for a user to ever supply their own version of `GoogleCredentials`.